dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facilities.repo.ts

```diff
@@ -402,4 +403,41 @@ export class TfmFacilitiesRepo {
       throw error;
     }
   }
+
+  /**
+   * Delete a portal amendment for a facility in the database
+   * @param deletePortalFacilityAmendment.facilityId - The facility id
+   * @param deletePortalFacilityAmendment.amendmentId - The amendment id
+   * @param deletePortalFacilityAmendment.auditDetails - The audit details
+   * @returns The delete result
+   */
+  public static async deletePortalFacilityAmendment({
```

Comment:
I think given the name of this function, we should make sure we filter on `Portal` amendment types so we only delete Portal amendments. (Or the alternative is to allow us to rename to `deleteFacilityAmendment` and allow either `Portal` or `Tfm` types to be deleted) :)